### PR TITLE
feat(hadoop): upgrade to hadoop 3.2.1

### PR DESCRIPTION
**Pull request by lyogev**

### CODE REVIEW

**Comment by RonBarabash** [comment]
@lyogev ❤️

File: scripts/docker_publish_dev.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+tag=$([ ! "${TRAVIS_PULL_REQUEST}" = "false" ] && echo "${TRAVIS_PULL_REQUEST}" || echo "${TRAVIS_BRANCH}")
```
Comment (liebstein) [inline scripts/docker_publish_dev.sh:4]: You don't want tag to be created if there's no PR?

Comment (lyogev) [reply scripts/docker_publish_dev.sh:4]: I do want it created but with the branch name

Comment (liebstein) [reply scripts/docker_publish_dev.sh:4]: So don't we need to add parentheses? just to make it clear?
```tag=$(([ ! "${TRAVIS_PULL_REQUEST}" = "false" ] && echo "${TRAVIS_PULL_REQUEST}") || echo "${TRAVIS_BRANCH}")```

Comment (lyogev) [reply scripts/docker_publish_dev.sh:4]: Oh, OK I can try... but it's bash :) not sure if it will work as expected

Comment (liebstein) [reply scripts/docker_publish_dev.sh:4]: So don't. Maybe it was just hard for me to understand

**Pull request merged**